Index: LICENSE
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/LICENSE b/LICENSE
new file mode 100644
--- /dev/null	(date 1751290612813)
+++ b/LICENSE	(date 1751290612813)
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2025 kigawa
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<project version=\"4\">\n  <component name=\"VcsDirectoryMappings\">\n    <mapping directory=\"\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/keruta-admin\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/keruta-admin/keruta-doc\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/keruta-agent\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/keruta-api\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/keruta-coder-template\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/keruta-doc\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/keruta-executor\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/kigawa-net-k8s\" vcs=\"Git\" />\n  </component>\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
--- a/.idea/vcs.xml	(revision 29724aebdec62fd02db5cb31ee6ee754d5b8f762)
+++ b/.idea/vcs.xml	(date 1766644326528)
@@ -10,5 +10,6 @@
     <mapping directory="$PROJECT_DIR$/keruta-doc" vcs="Git" />
     <mapping directory="$PROJECT_DIR$/keruta-executor" vcs="Git" />
     <mapping directory="$PROJECT_DIR$/kigawa-net-k8s" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/kigawa-net-k8s/fonsole-doc" vcs="Git" />
   </component>
 </project>
\ No newline at end of file
