Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<project version=\"4\">\n  <component name=\"VcsDirectoryMappings\">\n    <mapping directory=\"\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/keruta-admin\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/keruta-admin/keruta-doc\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/keruta-agent\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/keruta-api\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/keruta-coder-template\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/keruta-doc\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/keruta-executor\" vcs=\"Git\" />\n    <mapping directory=\"$PROJECT_DIR$/kigawa-net-k8s\" vcs=\"Git\" />\n  </component>\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
--- a/.idea/vcs.xml	(revision 4860e4ed22f8d0b1d0f27a814cfdc9156fe10688)
+++ b/.idea/vcs.xml	(date 1766103642939)
@@ -4,7 +4,6 @@
     <mapping directory="" vcs="Git" />
     <mapping directory="$PROJECT_DIR$/keruta-admin" vcs="Git" />
     <mapping directory="$PROJECT_DIR$/keruta-admin/keruta-doc" vcs="Git" />
-    <mapping directory="$PROJECT_DIR$/keruta-agent" vcs="Git" />
     <mapping directory="$PROJECT_DIR$/keruta-api" vcs="Git" />
     <mapping directory="$PROJECT_DIR$/keruta-coder-template" vcs="Git" />
     <mapping directory="$PROJECT_DIR$/keruta-doc" vcs="Git" />
Index: .idea/artifacts/client_jvm_dev.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><component name=\"ArtifactManager\">\n  <artifact type=\"jar\" name=\"client-jvm-dev\">\n    <output-path>$PROJECT_DIR$/ktcp/client/build/libs</output-path>\n    <root id=\"archive\" name=\"client-jvm-dev.jar\" />\n  </artifact>\n</component>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/artifacts/client_jvm_dev.xml b/.idea/artifacts/client_jvm_dev.xml
--- a/.idea/artifacts/client_jvm_dev.xml	(revision 4860e4ed22f8d0b1d0f27a814cfdc9156fe10688)
+++ b/.idea/artifacts/client_jvm_dev.xml	(date 1766103642895)
@@ -1,6 +1,8 @@
 <component name="ArtifactManager">
   <artifact type="jar" name="client-jvm-dev">
     <output-path>$PROJECT_DIR$/ktcp/client/build/libs</output-path>
-    <root id="archive" name="client-jvm-dev.jar" />
+    <root id="archive" name="client-jvm-dev.jar">
+      <element id="module-output" name="keruta.ktcp.client.jvmMain" />
+    </root>
   </artifact>
 </component>
\ No newline at end of file
